sections/4-development.tex

```diff
@@ -5,30 +5,32 @@ \section{Development Model}
 The \sunpypkg package is hosted on \github and uses \code{git}\footnote{\url{https://git-scm.com/}} as its distributed version control software.
 The entire code base is publicly available and anyone can suggest changes through pull requests.
 Since the code base is licensed under a permissive 2-clause BSD license\footnote{\url{https://opensource.org/licenses/BSD-2-Clause}}, anyone can redistribute, improve, repackage or use it in a closed environment as long as they credit the SunPy developers and redistribute the license.
-In order to maintain high quality code, every contribution must satisfy the following requirements:
+In order to maintain high quality code, every contribution is reviewed and must satisfy the following requirements:
 \begin{enumerate}
     \item Code and documentation must follow widely used style guides (e.g. PEP 8\footnote{\url{https://www.python.org/dev/peps/pep-0008/}}).
-    \item All new features must be accompanied with documentation, including code comments, formal documentation, and gallery examples.
+    \item All new features must be accompanied with documentation, including code comments, formal documentation such as \python docstrings and a guide, if appropriate, as well as gallery examples.
     \item Contributed code must provide unit tests which cover the majority of the new functionality.
-    \item All code must be within scope, and it must be reviewed and accepted by at least two members of the developer community before it is merged into the code base\footnote{This requirement is relaxed for bug fixes or small documentation changes.}.
+    \item All code must be within scope, and must be approved by at least two members of the developer community before it is merged into the code base\footnote{This requirement is relaxed for bug fixes or small documentation changes.}.
 \end{enumerate}
 These requirements are imposed on every pull request regardless of contributor.
 
-As of version 1.0, \sunpypkg consists of 48,427 lines of code\footnote{This number includes documentation and comments.
-There are 30,906 lines of pure code.} contributed by 123 unique contributors over 11,659 \git commits.
+As of Version 1.0, \sunpypkg consists of 48,427 lines of code\footnote{This number includes documentation and comments.
+There are 30,906 lines of code if documentation and comments are not counted.} contributed by 123 unique contributors over 11,659 commits.
```

Comment:
I think a word might be missing here ' As of Version 1.0, \sunpypkg consists of 48,427 lines of code contributed by 123 unique contributors _with_ over 11,958 commits

Maybe I'm reading it wrong though?